examples/mspm0g3507/src/bin/i2c.rs

```diff
@@ -0,0 +1,37 @@
+//! Example of using blocking I2C
+//!
+//! This uses the virtual COM port provided on the LP-MSPM0G3507 board.
+
+#![no_std]
+#![no_main]
+
+use defmt::*;
+use embassy_executor::Spawner;
+use embassy_mspm0::i2c::{BusSpeed, ClockSel, Config, I2c};
+use {defmt_rtt as _, panic_halt as _};
+
+const ADDRESS: u8 = 0x6a;
```

Comment:
What device is this by chance? (I don't think the BSL connection for the launchpad boards since the XDS110 runs its own I2C in controller mode.

Reply:
While it would be ideal to use a common device for the example, none of the launchpad boards have a built-in i2c device to play with...

Is this a specific board or is it some internal thing you can't release (if that is the case, we can always change the example in the future to something publicly accessible).